scripts/zones/Batallia_Downs/mobs/Skirling_Liger.lua

```diff
@@ -0,0 +1,49 @@
+-----------------------------------
+-- Area: Batallia Downs (105)
+--  MOB: Skirling Liger
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+    if (mob:getMod(MOD_REGAIN) ~= 0) then
+        mob:setMod(MOD_REGAIN,0);
+    end
+end;
+
+-----------------------------------
+-- onMobFight
+-----------------------------------
+
+function onMobFight(mob, target)
+    mob:setMod(MOD_REGAIN, 50);
```

Comment:
that'll needlessly run every tick. if he's only supposed to have regain while in battle, set in in onMobEngaged() and take away in onMobDisengage() uing the precise amount with addMod()/delMod() instead of the ~0 stuff

only IF that is so though (I didn't look them open before posting, and do not recall how it works)